### ISSUE TITLE
Event packing leaks a variable into the current context

### ISSUE THREAD

**Issue by smarx** [ethereum/vyper#1688]
```python
Foo: event({a: bytes[25]})

@public
@constant
def test() -> string[7]:
    log.Foo("testing")
    return _log_pack_var_6_12 # returns "testing"
```

**Comment by fubuloubu** [comment]
Potentially solved by #1601

**Comment by charles-cooper** [comment]
confirmed fixed in master (4666a872acae10efed0f9c349e9359e686bd3a08)
```
$ vyper -f ir /dev/stdin <<EOF
> Foo: event({a: bytes[25]})
> 
> @public
> @constant
> def test() -> string[7]:
>     log.Foo("testing")
>     return _log_pack_var_6_12 # returns "testing"
> EOF
Error compiling: /dev/stdin
vyper.exceptions.VariableDeclarationException: line 7:11 Undeclared variable: _log_pack_var_6_12
     6     log.Foo("testing")
---> 7     return _log_pack_var_6_12 # returns "testing"
------------------^
```
